src/sentry/buffer/base.py

```diff
@@ -69,6 +69,8 @@ def incr(
             }
         )
 
+    # TODO: `partition` is unused, remove after a deploy
```

Comment:
```suggestion
    # TODO(@asottile-sentry): `partition` is unused, remove after a deploy
```

Reply:
meh, I have the followup branch already ready :D